api/tests/python/nro_services/test_nro_utils.py

```diff
@@ -14,7 +14,24 @@
 ]
 
 
+compress_name_test_data = [
```

Comment:
Consider moving the test data closer to the test, so that seeing the test cases and the tests in a single screen pane is easier.